### PR TITLE
feat: export all types of sdks

### PR DESCRIPTION
**Pull request by akira02**
## Summary

Export types from all sdks for convenience

## Related Links

**Asana**:

## Checklist

- [ ] Pasted Asana link.
- [ ] Tagged labels.

### CODE REVIEW

**Review by mordochi — APPROVED** [review]
Thank you so much!

File: packages/blocto-sdk/src/index.d.ts [hunk 1]
```diff
@@ -26,8 +27,10 @@ export {
   SolanaProviderInterface,
   AptosProviderConfig,
   AptosProviderInterface,
-  AptosSignMessagePayload,
-  AptosSignMessageResponse,
+  // Keep above types for backward compatibility
+  AptosTypes,
+  SolanaTypes,
+  EthereumTypes,
```
[the remark this answers is not in the document]
Comment (mordochi) [reply packages/blocto-sdk/src/index.d.ts:33]: Oh yeah agree. But I guess we can change the name in `./providers/types/aptos.d` directly, or it's a bit inconsistent for the experience of importing the types from Blocto SDK (I mean some can be imported directly and some needs to be imported by the chain type)

[the remark this answers is not in the document]
Comment (akira02) [reply packages/blocto-sdk/src/index.d.ts:33]: I think it's fine. `ProviderConfig` or `ProviderInterface` can imported directly beacuse they used in initial part, and other detail type keep in `[Chain]Types`.

[the remark this answers is not in the document]
Comment (mordochi) [reply packages/blocto-sdk/src/index.d.ts:33]: Ok no problem 
Thanks!

**Pull request merged**